### DISCUSSION TITLE
JKBMS PB2A16S20P hw 15.xx supported? wich yaml?

### DISCUSSION THREAD

**Discussion by man55** [syssi/esphome-jk-bms#558] [Q&A] [answered]
Hi,
I am waiting for the JKBMS to be delivered and in the meantime preparing to install the HA integration.

Is this PB2A16S20P supported with the latest firmware version 15?
If so, which yaml file should I choose for BLE connection? There are so many of them and I am a bit confused )).
Tnx.

**Comment by syssi** [comment, marked as answer]
Please use the  esp32-ble-v14-example.yaml. I don't expect much differences between v14 and v15.

**Reply by man55** [reply]
Thank you.

I read some reviews that the Bluetooth connection may work unstable. 
Is there a BLE connection sensor to visually control the quality of the connection in the HA dashboard and not constantly go into the logs?

**Reply by syssi** [reply]
Yes, there are two sensors. One monitors the availability of the ESPHome node. The second the responsiveness of the BMS.
